src/fsharp/FSharp.Core/FSharp.Core.fsproj

```diff
@@ -211,9 +212,41 @@
                                                 '$(TargetFramework)' == 'XNA\5.0' OR 
                                                 '$(TargetFramework)' == 'sl3-wp' "/>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'coreclr'">
+    <Reference Include="$(FSharpSourcesRoot)\..\packages\System.Collections\4.0.10-beta-22809\ref\any\System.Collections.dll" />
```

Comment:
I also have a project.json file too, it was necessary to get the nugget downloads working, I still have to learn what the syntax differences are for referencing them that way.  To be honest I was focused for now on getting something working, where I could easily diagnose what was going on.  I fully expect to modify things moving forward.